libs/input-mapping/src/Helper/RealDevStorageTableRewriteHelper.php

```diff
@@ -37,22 +48,24 @@ private function rewriteSourceBranchId(
         string $source,
         ClientWrapper $clientWrapper,
         LoggerInterface $logger
-    ): string {
+    ): array {
         if ($clientWrapper->hasBranch() && $clientWrapper->getBranchClient()->tableExists($source)) {
             $logger->info(sprintf(
                 'Using dev input "%s" from branch "%s" instead of main branch "%s".',
                 $source,
                 $clientWrapper->getBranchId(),
                 $clientWrapper->getDefaultBranch()['branchId']
             ));
-            return (string) $clientWrapper->getBranchId();
+            $tableInfo = $clientWrapper->getBranchClient()->getTable($source);
+            return [$tableInfo, $clientWrapper->getBranchId()];
         }
         $logger->info(sprintf(
             'Using fallback to default branch "%s" for input "%s".',
             $clientWrapper->getDefaultBranch()['branchId'],
             $source,
         ));
         // use production table
-        return $clientWrapper->getDefaultBranch()['branchId'];
+        $tableInfo = $clientWrapper->getBasicClient()->getTable($source);
+        return [$tableInfo, $clientWrapper->getDefaultBranch()['branchId']];
```

Comment:
Toto je to hlavní, kvůli čemu se to celý dělá: 

![image](https://github.com/keboola/platform-libraries/assets/4319320/b9965aa4-e61c-4c51-968b-00dfd9aa43d0)

Reply:
Místo basicClient by se tady měl dát getDefaultBranch client, aby to nebylo úplně jednoduchý, ale to nebylo v době psaní tohhle textu ještě releasnutý https://github.com/keboola/storage-api-php-client/pull/1124/files#diff-34a77dd53b8861c383400fc88019eb8a04f54f97e17a3bda19221486e2632b2aR189